README.md

```diff
@@ -27,11 +27,11 @@ If no feature is specified, the crate will automatically select the platform-app
 
 The following platform and backend combinations are supported and tested in CI:
 
-| Platform  | Vulkan | OpenGL | Metal |
-|-----------|--------|--------|-------|
-| Linux x86 | ✅     | ✅     | ❌    |
-| Linux ARM | ✅     | ✅     | ❌    |
-| macOS ARM | 🟨     | 🟨     | 🟨    |
+| Platform  | Vulkan        | OpenGL | Metal |
+| --------- | ------------- | ------ | ----- |
+| Linux x86 | ✅            | ✅     | ❌    |
+| Linux ARM | ✅            | ✅     | ❌    |
+| macOS ARM | 🟨 (MoltenVK) | ❌     | 🟨    |
```

Comment:
Just to clarify:
If vulcan just under the hood uses metal. Why would you want to use it then?
It just seems like performance overhead..

Should we be more explcit in not recomending this, or is the adapter level quite thin and thus not actually a problem? 🤔

Reply:
The overhead is not too bad, but there of course will be a bit. Someone on macos would effectively only opt for the vulkan backend in case a metal option was lacking.

Reply:
I think I have reworded the PR content to reflect this, could you look over if this is accurate?